### ISSUE TITLE
The link is broken on PyPI --> quickstatements-client

### ISSUE THREAD

**Issue by lubianat** [cthoyt/quickstatements_client#7]
Hey Charlie, 

Link to github seems to be broken on PyPI: 

```
<li>
      <a class="vertical-tabs__tab vertical-tabs__tab--with-icon vertical-tabs__tab--condensed" href="https://github.com/cthoyt/quickstatements-client" rel="nofollow">
        

<i class="fab fa-github" aria-hidden="true"></i>Source Code
      </a>
    </li>
```

There is a **-** where it should be a **_**, I guess. 

https://pypi.org/project/quickstatements-client/

Not sure how to fix it, though

cheers

**Comment by cthoyt** [comment]
Seems like this was already fixed in #6, we just need a new release

**Comment by cthoyt** [comment]
New release made! Thanks for pointing this out!
